.github/workflows/release.yml

```
@@ -0,0 +1,73 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  release:
+
+    runs-on: ubuntu-latest
+
+    container:
+      image: analysiscenter1/ds-py3:cpu
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build docs
+      run: |
+        export LC_ALL="C"
+        pip3 install --upgrade sphinx sphinx-rtd-theme
+        git checkout --orphan gh-pages
+        rm -rf *
+        git fetch
+        git checkout origin/master -- batchflow docs
```

Replace last 3 lines with
```suggestion
        git reset
        git clean -fdx
        git checkout ${{ github.sha }} -- batchflow docs
```